src/cart/crt_init.c

```diff
@@ -235,6 +235,9 @@ prov_data_init(struct crt_prov_gdata *prov_data, crt_provider_t provider,
 		}
 	}
 
+	if (set_sep)
+		D_WARN("Scalable endpoint mode not supported. Unset CRT_CTX_SHARE_ADDR\n");
```

Comment:
I wanted for NLT/CI to fail if it still sets sep somewhere as it might be running the same test twice, one with and one without SEP. Printing warning at least would fail NLT if this was the case